### PR TITLE
change habdroid to openhab app

### LINKED ISSUE #283: Is the app still named HabDroid?
Since it is called openHAB in the Play Store shouldn't we change "habdroid" to "openhab app" in all strings.xml?

### PR DESCRIPTION
**Pull request by mueller-ma**
Fix #283 
What about Spanish, French, etc. strings? Should I remove the wrong translations?

Done in:
- [x] English
- [x] German
- [x] Spanish
- [ ] French
- [x] Japanese
- [x] Lithuanian
- [x] Dutch

### CODE REVIEW

**Comment by digitaldan** [comment]
Thanks for this, it would be great to change the strings in all the translations.  If in doubt I would post to the community forum for help.

**Comment by mueller-ma** [comment]
@mgarciate @sosandroid @belovictor @naofum @jvitkauskas @SJoWie80 @friek 
Can you update your translation?

**Comment by jvitkauskas** [comment]
Please see PR https://github.com/mueller-ma/openhab.android/pull/1 for updated Lithuanian translation. Also see my comment, you have a mistake in "settings_openhab_screentimeroff_summary" English translation.

**Comment by digitaldan** [comment]
This is awesome, can't wait to merge when we get them all.

**Comment by mgarciate** [comment]
Hi @mueller-ma, check it out mueller-ma#2

**Comment by mueller-ma** [comment]
Dutch is in #304

**Comment by digitaldan** [comment]
So close !!!

File: mobile/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -38,11 +38,11 @@
     <string name="settings_openhab_sslclientcert_howto">SSL client certificate how-to</string>
     <string name="settings_openhab_sslclientcert_howto_url" translatable="false">https://github.com/openhab/openhab.android/blob/master/KeyAuth.md</string>
     <string name="settings_openhab_screentimeroff">Disable display timer</string>
-    <string name="settings_openhab_screentimeroff_summary">Disable display switch off timer when HABDroid is running</string>
+    <string name="settings_openhab_screentimeroff_summary">Disable display switch off timer when openHAB is running</string>
     <string name="settings_openhab_demomode">Demo mode</string>
-    <string name="settings_openhab_demomode_summary">Run HABDroid in demo mode</string>
-    <string name="settings_openhab_theme">HABDroid Theme</string>
-    <string name="settings_openhab_appversion">HABDroid Version</string>
+    <string name="settings_openhab_demomode_summary">Run openHAB App in demo mode</string>
```
Comment (digitaldan) [inline mobile/src/main/res/values/strings.xml]: Sorry for bringing this up at the last minute, but when testing this "Run openHAB App" sounds broken.  This should  probably  "Run in demo mode"

File: mobile/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -54,7 +54,7 @@
     <string name="title_voice_widget">openHAB Voice Commands</string>
     <string name="info_voice_input">"openHAB, at your command!"</string>
     <string name="info_voice_recognized_text">"Recognized command: %1$s"</string>
-    <string name="info_demo_mode">HABDroid is running in demo mode. To connect to your openHAB please go to settings menu.</string>
+    <string name="info_demo_mode">openHAB App is running in demo mode. To connect to your openHAB server please go to settings menu.</string>
```
Comment (digitaldan) [inline mobile/src/main/res/values/strings.xml]: again, "openHAB App"  sounds broken.  I would change to "Running in demo mode."

File: mobile/src/main/res/values/strings.xml (hunk 1 above)
Comment (mueller-ma) [inline mobile/src/main/res/values/strings.xml]: @digitaldan right. it changed it

File: mobile/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -40,7 +40,7 @@
     <string name="settings_openhab_screentimeroff">Disable display timer</string>
     <string name="settings_openhab_screentimeroff_summary">Disable display switch off timer when openHAB is running</string>
     <string name="settings_openhab_demomode">Demo mode</string>
-    <string name="settings_openhab_demomode_summary">Run openHAB App in demo mode</string>
+    <string name="settings_openhab_demomode_summary">Running in demo mode</string>
```
Comment (digitaldan) [inline mobile/src/main/res/values/strings.xml]: This should be "Run in demo mode"

**Comment by digitaldan** [comment]
I think we should go ahead and just merge this, I'll shoot for a release by Sunday night.

**Comment by mueller-ma** [comment]
Maybe add a todo note to the french strings

**Comment by digitaldan** [comment]
Done!

**Comment by lolodomo** [comment]
I will take a look to the french translation and provide updates if necessary.

**Comment by mueller-ma** [comment]
French is in #377

**Pull request merged**